scripts/visual-active-effects.mjs

```diff
@@ -36,17 +36,32 @@ export class VisualActiveEffects extends Application {
       // Set up the various text (intro and content).
       const desc = entry.effect.flags["dfreds-convenient-effects"]?.description;
       const data = entry.effect.flags[MODULE]?.data ?? {};
+      
+      // Get the effect rollData to populate enrichers within the descriptions.
+      let rollData = {};
+      if (entry.effect.origin) {
+        const origin = await fromUuid(entry.effect.origin);
+        if (origin && typeof origin.getRollData === 'function') {
```

Comment:
```suggestion
        if (origin && (typeof origin.getRollData === "function")) {
```

Reply:
I simplified this down to `if (typeof origin?.getRollData === "function") {` but let me know if you're not a fan of that style.